task-json/q3.js

```diff
@@ -0,0 +1,48 @@
+function generateArrayOfNumbers(arrayLength){
+    const arr=new Array(arrayLength);
+    
+
+arr.fill().map((value,indx)=>(arr[indx]=Math.floor(Math.random()*10)));

```

Comment:
use it using normal for loop

Reply:
u mean not using fill and map instead i use for loop ?